neqo-transport/src/cc.rs

```diff
@@ -220,27 +227,17 @@ impl CongestionControl {
     }
 
     #[must_use]
-    fn in_congestion_recovery(&mut self, sent_time: Instant, packet_state: PacketState) -> bool {
+    fn in_congestion_recovery(&mut self, sent_time: Instant) -> bool {
         match self.congestion_recovery_start_time {
```

Comment:
self.congestion_recovery_start_time.map_or(false, |t| sent_time <= t)

Reply:
Going to pass, I don't think this improves comprehension.